scripts/run.sh

```diff
@@ -17,25 +17,35 @@ lease
 # Get node information
 kubectl get nodes
 
-# Get Helm up to date and checkout branch if needed
-echo "Adding repo ${HELM_REMOTE_REPO}"
-helmc repo add deis "${HELM_REMOTE_REPO}"
-echo "Get helm up to date"
-helmc up
-cd "${DEIS_CHART_HOME}" || exit
-echo "Checking out ${WORKFLOW_BRANCH} for ${WORKFLOW_CHART}"
-git checkout "${WORKFLOW_BRANCH}"
-helmc fetch "deis/${WORKFLOW_CHART}"
-
 # Clean cluster if needed
 echo "Cleaning cluster if needed"
 clean_cluster
 
-# Install $WORKFLOW_CHART
-echo "Generate manifests from templates"
-helmc generate "${WORKFLOW_CHART}"
-echo "Installing chart ${WORKFLOW_CHART}"
-helmc install "${WORKFLOW_CHART}" &> /dev/null
+if [ -n "$USE_KUBERNETES_HELM"]
```

Comment:
shellcheck probably just wants a space between the `"` and the `]`, both here and below.